src/NPlaylist/WPL/WplDeserializer.cs

```diff
@@ -0,0 +1,78 @@
+using System;
+using System.IO;
+using System.Xml.Serialization;
+
+namespace NPlaylist.Wpl
+{
+    public class WplDeserializer : IPlaylistDeserializer<WplPlaylist>
+    {
+        public WplPlaylist Deserialize(string input)
+        {
+            if (input == null)

```

Comment:
I think it would be better to check if 
```suggestion
            if(string.IsNullOrWhitespace(input))
```